resources/styles/less/components/loopobject.less

```diff
@@ -0,0 +1,74 @@
+/** *** *** *** *** *** *** *** *** **/
+/** ***  L O O P   O B J E C T  *** **/
+/** *** *** *** *** *** *** *** *** **/
+
+.loop_object {
+    padding-top: 10px;
+    padding-bottom: 10px;
+}
+
+.loop_object_content>p {
+	margin: 0px;
+}
+
+.loop_object_content {
+	margin-bottom:5px;
+}
+
+.loop_object.float-left {
+	margin-right:10px;
+}
+
+.loop_object.float-right {
+	margin-left:10px;
+}
+
+.loop_object_content .float-left, .loop_object_content .float-right {
+	float:none;
+}
+
+.loop_object_name, .loop_object_number, .loop_object_title_seperator {
+	font-weight:bold;
+}
+
+.loop_object_footer {
+	border-left: 2px solid @accent-color;
+	padding-left: 5px;
+	line-height: 18px;
+}
+
+.loop_object_description, .loop_object_title_content {
+	line-break: loose;
+	-ms-line-break: loose;
+}
+
+.loop_object_icon .ic {
+    font-size: 18px;
+    vertical-align: middle;
+}
+
+.list_of_figures img {
+	max-width: 100%;
+	height: auto;
+}
+
+.loop_object_title_content {
+	font-weight: bold;
+}
+
+.list_of_objects tr td:first-child {
+	min-width: 105px;
+}
+
+.list_of_figures tr td:nth-child(2) {
+	min-width: 75px;
+}
+
+.table-hover tr:hover {
```

Comment:
layout/content allgemein? hier wird nur die hoverfarbe von tabellen definiert.. das wird zwar bislang nur in den objects benutzt, hat aber effekt auf alles.

Reply:
nein, nein. ext.bootstrap, da es bootstrap klassen sind!